pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
+    "astunparse",
+    "numpy",
+    "ninja",
+    "pyyaml",
+    "setuptools",
+    "cmake",
+    "cffi",
+    "typing_extensions",
+    "future",
+    "six",
+    "requests",
+    "dataclasses",
+]
+build-backend = "setuptools.build_meta:__legacy__"
+
+
+[tool.black]
+# Uncomment if pyproject.toml worked fine to ensure consistency with flake8
+# line-length = 120
+skip-string-normalization = true
```

Comment:
line-length:
Motivation of this PR. flake8 is already specified 120 [here](https://github.com/pytorch/pytorch/blob/ba27ee9e8fc57f7509d2bf0c0be73510802806c0/.flake8#L3), and is already widely applied. So I thought that it is better to adjust black configuration.

skip-string-normalization:
This option can be removed. Some Python files use single quote for string, so I wanted to avoid accidental changes. However currently these files are not target of black.
https://github.com/pytorch/pytorch/blob/ba27ee9e8fc57f7509d2bf0c0be73510802806c0/torch/nn/modules/conv.py#L618

target-version:
The lint error in test/onnx/test_pytorch_onnx_caffe2.py is caused by black's default target-version "per-file auto-detection".
Trailing comma after function keyword arguments is supported from Python 3.6. This means that black working as unexpected old Python version (3.5<=) in test/onnx/test_pytorch_onnx_caffe2.py.
It is better to specify target-versoin to avoid unexpected behavior like this.

Reply:
I would like to suggest keeping the line length to black’s default. Because (1) black spent time to decide on it with good reasons (readability, accessibility and convenience - 120 is really wide for side by side panes) https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length (2) flake’s 120 is the max allowed width, not necessarily a desirable one